mods/cnc/rules/aircraft.yaml

```diff
@@ -2,6 +2,8 @@ TRAN:
 	Inherits: ^Helicopter
 	Valued:
 		Cost: 750
+    GivesExperience:
+        Experience: 750
```

Comment:
Did you mean for all of these to be x 100 to balance out the change in the trait?